SettingsList.py

```diff
@@ -2430,22 +2430,16 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             True: {'settings': ['shuffle_weird_egg']},
         },
     ),
-    Setting_Info(
+    Combobox(
         name           = 'dungeon_shortcuts',
-        type           = list,
-        gui_text       = 'Dungeon Boss Shortcuts',
-        gui_type       = "MultipleSelect",
+        gui_text       = 'Dungeon Boss Shortcuts ',
+        default        = 'off',
         choices        = {
-            'deku_tree': 'Deku Tree',
-            'dodongos_cavern': 'Dodongo\'s Cavern',
-            'jabu_jabus_belly': 'Jabu Jabu\'s Belly',
-            'forest_temple': 'Forest Temple',
-            'fire_temple': 'Fire Temple',
-            'water_temple': 'Water Temple', # doesn't do anything, but added to prevent confusion
-            'shadow_temple': 'Shadow Temple',
-            'spirit_temple': 'Spirit Temple'
+            'off':       'Off',
+            'choice':    'Choice dungeons',
+            'all':	  'All dungeons',
+            'random': 'Random dungeons'
```

Comment:
Nitpick: Spacing here is seemingly random, should align all of the options. Looks like the actual shortcut choices weren't aligned either so a good opportunity to do that as well.

Reply:
Fix : https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/b3958c886644d5d247a9adc47cb33b4e9a50bafb